docs/concepts/architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Using Arcus Event Grid Sidecar
+---
+
+# Architecture
+It's a hosted REST API that allows you to POST events to which will be forwarded to an Azure Event Grid Topic of choice.
```

Comment:
I was thinking on how the sentence could start with something else then 'it' but I can't think of anything.
I think it will be more fluently readable when it doesn't start with it I guess.

Reply:
`We provide a hosted REST API that allows you to POST events to which will be forwarded to an Azure Event Grid Topic of choice.`?